### ISSUE TITLE
heroku instructions?

### ISSUE THREAD

**Issue by geemus** [interagent/pliny#263]
Might be nice to have some extra heroku-specific instructions. In particular, if I recall, the bin/setup stuff bombs out because it tries to do stuff heroku-postgres doesn't provide permissions for (drop-table maybe?). It would be great if this worked somehow in the heroku context, but barring that a small section in the instructions would be nice. Also possible I'm just totally missing something here.

**Comment by gudmundur** [comment]
Yeah, I agree. I'll take a stab at better setup instructions soon. I feel we're mostly lacking on the documentation front right now.

**Comment by geemus** [comment]
Yeah. I had thought I might try to take a stab myself, but was concerned I might just be doing it wrong. So I thought I would at least point out where I got caught up in case it wasn't on your radar. Thanks!
